apps/system/js/app_window.js

```diff
@@ -926,6 +923,13 @@
         this[componentName] = null;
       }
 
+      for (var propertyName in this.constructor.SUB_MODULES) {
+        if (this[propertyName] && this[propertyName].stop) {
+          this[propertyName].stop();
+        }
+        this[propertyName] = null;
+      }
+
```

Comment:
Actually, I'm not sure where we moved BrowserContentMenu in the dependency chain.